frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableRow.tsx

```diff
@@ -110,28 +111,25 @@ const PipelinesTableRow: React.FC<PipelinesTableRowProps> = ({
         </Tr>
         {isExpanded && (
           <PipelinesTableExpandedRow
-            // Upload a new version will update the pipeline default version
-            // Which could trigger a re-render of the versions table
-            key={pipeline.default_version?.id}
+            // Upload a new version will update the totalSize (version length)
+            // Which will trigger a re-render of the versions table
+            key={`${pipeline.pipeline_id}-expanded-${totalSize}`}
```

Comment:
Wait, why does this need to happen? I get this logic is coming from Versions -- but what? @DaoDaoNoCode why did you need to do this?

Reply:
That is not making any sense to me lol -- why wouldn't your data just update inline? This is React...